src/NovelRT.Interop/Timing/NrtStepTimer.cpp

```diff
@@ -3,150 +3,138 @@
 
 #include <NovelRT.Interop/NrtErrorHandling.h>
 #include <NovelRT.Interop/Timing/NrtStepTimer.h>
-#include <NovelRT/NovelRT.h>
+#include <NovelRT/Timing/StepTimer.h>
+#include <NovelRT/Utilities/Event.h> // The order is important for some reason.
```

Comment:
Can you clarify the comment here?

Reply:
I left that out by accident, so it's gone now.